amqtt/broker.py

```diff
@@ -495,9 +497,18 @@ async def _handle_client_session(
 
         self.logger.debug(f"{client_session.client_id} Start messages handling")
         await handler.start()
-        self.logger.debug(f"Retained messages queue size: {client_session.retained_messages.qsize()}")
+
+        # publish messages that were retained because the client session was disconnected
+        self.logger.debug(f"Offline messages queue size: {client_session.retained_messages.qsize()}")
```

Comment:
Curious why rename retained to offline here but not everywhere?  I prefer the retained terminology as that's what the MQTT spec uses.